audio-worklet/basic/audio-worklet-node-options/basic-processor.js

```diff
@@ -0,0 +1,66 @@
+
+// https://en.wikipedia.org/wiki/Sawtooth_wave
+const sawtooth = (f, t) => {
```

Comment:
No abbreviation on arguments:
https://google.github.io/styleguide/jsguide.html#naming-rules-common-to-all-identifiers

Reply:
Ditto for other functions below.